build/rmutil/Makefile

```diff
@@ -12,15 +12,27 @@ SRCDIR=$(ROOT)/deps/RedisModulesSDK/rmutil
 CC=gcc
 
 CC_FLAGS=\
-	-g \
```

Comment:
Maybe:
```
ifeq ($(PROFILE),1)
override DEBUG=1
endif
```
This will also cause binary files go into `bin/linux-x64-debug` rather then into `bin/linux-x64-release`.
WDYT?

Reply:
@rafie the `DEBUG=1` will make us use `-O0 -DVALGRIND` which is not what we want to target when doing the profile. 
I would rather keep the logic as is, meaning we will have the artifact `bin/linux-x64-release-profile/redistimeseries.so`. 
WDYT?